### PR TITLE
Add list element component

### LINKED ISSUE #10: Add list element component
#### Description

The list element component for the "Übersicht"-page.

#### Screenshots

image

image

#### Tasks

Include specific tasks in the order they need to be done in. Include links to specific lines of code where the task should happen at.

- [x] Div with text, icon
- [x] Write story for storybook

### PR DESCRIPTION
**Pull request by Maurizio-F**
The list element component for the "Übersicht"-page.

fixes #10 

heroku storybook preview:
https://movie-night-maurizio-f--qbiy0k.herokuapp.com/storybook/?path=/story/component-list--list-element-with-share

### CODE REVIEW

File: src/app/components/List/List.module.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.listElement {
+  background-color: var(--primary-color);
+  color: var(--secondary-text-color);
+  display: grid;
+  grid-template-columns: 10em 1fr;
+  border-top-left-radius: 10px;
```
Comment (JGieseWeb) [inline src/app/components/List/List.module.css]: Why use px here ? :D Even if the radius should not change, better em, right?

**Review by JGieseWeb — APPROVED** [review]

File: src/app/components/List/List.module.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
+.listElement {
+  background-color: var(--primary-color);
```
Comment (Morten-Moeller) [inline src/app/components/List/List.module.css]: Primary-color seams a bit off for a background color. Consider to create another variable for background color.

File: src/app/components/List/List.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+export default {
+  title: "Component/List",
+  component: List,
+};
+
+export const ListElementWithShare = (): JSX.Element => {
+  return (
+    <div>
+      <List
```
Comment (Morten-Moeller) [inline src/app/components/List/List.stories.tsx]: So many entry's, i miss an array with a map method.

**Review by Morten-Moeller — COMMENTED** [review]
Just a few suggestions. Happy coding.

**Review by lajanzen — APPROVED** [review]
I'd say the same thing like Jannik about the pixels, but apart from that it lgtm :)

**Review by daebert — APPROVED** [review]

File: src/app/components/List/List.module.css (hunk 2 above)
Comment (Maurizio-F) [reply src/app/components/List/List.module.css]: Thanks for the tip, I have generally reconsidered my naming in global.css.

**Pull request merged**